__tests__/app.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json");
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
```

Comment:
Good work implementing the 2 jest lifecycle hooks. You're successfully seeding the test database with test data only also.